app/Actions/Install/RequirementsChecker.php

```diff
@@ -27,7 +27,10 @@ public function check(array $requirements): array
 					foreach ($requirement_ as $requirement) {
 						$hasExtension = extension_loaded($requirement);
 						$results['requirements'][$type][$requirement] = $hasExtension;
-						$results['errors'] |= !$hasExtension;
+						// Note: Don't use the short-cut assignment `|=`;
+						// it silently converts the type to integer, because
+						// `|` is not the logical OR, but the bitwise OR.
```

Comment:
FFS.

Reply:
Yes. Actually, my IDE told me about that. But it was categorized as a "notice". ("The result of this operation might potentially not be what you expect") Haha! Hence, it was basically flying below my radar in the noise of the remaining 400 warnings and errors.